selfdrive/controls/lib/drive_helpers.py

```diff
@@ -65,6 +65,10 @@ def update_v_cruise(v_cruise_kph, buttonEvents, button_timers, enabled, metric):
         long_press = True
         break
 
+  # If set is pressed while overriding, reset cruise speed to vEgo
+  if enabled and gas_pressed and button_type == car.CarState.ButtonEvent.Type.decelCruise:
+    return int(round(clip(v_ego * CV.MS_TO_KPH, V_CRUISE_ENABLE_MIN, V_CRUISE_MAX)))
```

Comment:
Round() to whole numbers already returns an integerer
```suggestion
    return round(clip(v_ego * CV.MS_TO_KPH, V_CRUISE_ENABLE_MIN, V_CRUISE_MAX))
```

Reply:
It's a leftover from the python 2 days where it returned a float.